DevAcademy/Model/Feature.swift

```diff
@@ -0,0 +1,148 @@
+import Foundation
+
+struct Feature: Equatable{
```

Comment:
Jenom takova kosmeticka pripominka, pred oteviraci zavorky `{` je ve Swiftu zvyklost davat mezeru pro lepsi citelnost. 
Mas to na dost mistech v tomhle PR, tak jestli to budes fixovat tak na to mrkni vsude, at to mas jednotny. 😉